python/run_slam.py

```diff
@@ -0,0 +1,387 @@
+
+    slam_options_message.add_row(_header("-profile"), "[b color(4)]USAGE:[/]", 
+                            f"""[b]python run_slam.py [/]{_header('-profile')} {_argument('--profile_name')} <value>""")
+    slam_options_message.add_row("", f"{_argument('--profile_name')}", "A profile of options, among: [b blue][DRIVING, MOBILE_ROBOTS_OUTDOOR, INDOOR][/]")
+
+
+    console.print(rich.panel.Panel(
+        slam_options_message,
+        border_style="color(3)",
+        title="[b]SLAM OPTIONS[/]", 
+        title_align="left",
+        padding=(0, 1)
+    ))
+
+    # ###################################### #
+    # ########### OUTPUT OPTIONS ########### #
+    # ###################################### #
+
+    output_options_message = rich.table.Table.grid(padding=1)
+    output_options_message.add_column(no_wrap=True)
+    output_options_message.add_column(no_wrap=True)
+    output_options_message.add_column(no_wrap=True)
+
+    output_options_message.add_row("", "", "Specify the options for the outputs of the SLAM (trajectory, frames, etc...)")
+
+    output_options_message.add_row("[b color(4)]USAGE:[/]", "",
+                            f"""[b]python run_slam.py [/]{_argument('--output_dir')} <value>""")
+    output_options_message.add_row("", f"{_argument('--output_dir')}",
+                            f"The path on disk to the directory to output the trajectory and summaries")
+
+    console.print(rich.panel.Panel(
+        output_options_message,
+        border_style="color(3)",
+        title="[b]OUTPUT OPTIONS[/]", 
+        title_align="left",
+        padding=(0, 1)
+    ))
+
+
+# ################################################## #
+def parse_arguments()->Arguments:
+    """
+    Parses Arguments from the command line
+    """
+    parser = argparse.ArgumentParser(description="")
+    parser.print_help = print_help
+
+    # Config
+    parser.add_argument("-dataset", action='store_true')
+    parser.add_argument("-ply", action='store_true')
+    parser.add_argument("-dataset_config", action='store_true')
+
+    parser.add_argument("--name",type=str)
+    parser.add_argument("--root",type=str)
+    parser.add_argument("--dataset_path",type=str)
+
+    # Slam Options
+    parser.add_argument("-slam_config")
+    parser.add_argument("-profile")
+
+    parser.add_argument("--config_path",type=str)
+    parser.add_argument("--profile_name",type=str)
+
+    # Output and Execution Options
+    parser.add_argument("--output_dir",type=str)
+    parser.add_argument("--max_num_frames",type=int) # The maximum number of frames per sequence
+
+
+    # Parse Args 
+    parsed_args = parser.parse_args()
+
+    # Check which dataset option is selected
+    dataset_option : Optional[Arguments.DATASET_OPTION]= None
+    if parsed_args.dataset:
+        dataset_option = Arguments.DATASET_OPTION.dataset
+    elif parsed_args.ply:
+        dataset_option = Arguments.DATASET_OPTION.ply
+    elif parsed_args.config:
```

Comment:
There is no `config` option in the arguments.
Did you mean `dataset_config`?
```suggestion
    elif parsed_args.dataset_config:
```